sklearn/linear_model/coordinate_descent.py

```diff
@@ -826,9 +826,9 @@ class Lasso(ElasticNet):
         reasons, using ``alpha = 0`` with the ``Lasso`` object is not advised.
         Given this, you should use the :class:`LinearRegression` object.
 
-    fit_intercept : boolean
-        whether to calculate the intercept for this model. If set
-        to false, no intercept will be used in calculations
+    fit_intercept : boolean, optional, default True
+        Whether to calculate the intercept for this model. If set
+        to False, no intercept will be used in calculations
```

Comment:
you should state the default value inside parenthesis:
`fit_intercept : boolean, optional (default = True)`

Reply:
Hum actually there are a lot of inconsistencies about that...
And looking at the docstring, many optional parameters dont mention the default value. Maybe we should consider mention them to.

Reply:
So what should be the scope of this PR? The whole file ? and you suggest to harmonize with the (default = foo) style?